File: src/java/WebNinja/src/test/java/webninja/selenuim/AdminPage.java
package webninja.selenuim;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

public class AdminPage {
    private final WebDriver webDriver;

    public AdminPage(WebDriver webDriver) {
        this.webDriver = webDriver;
    }

    public ProjectForm showNewProjectForm() {
        webDriver.findElement(By.linkText("Add a Project")).click();
        return new ProjectForm(webDriver);
    }
}

File: src/dotnet/WebNinja/WebNinja/selenium/AdminPage.cs
using OpenQA.Selenium;

namespace WebNinja.selenium
{
    public class AdminPage : PageObject
    {
        public AdminPage(IWebDriver driver) : base(driver)
        {
        }

        public ProjectForm GotoProjectForm()
        {
            Driver.FindElement(By.LinkText("Add a Project")).Click();
            return new ProjectForm(Driver);
        }
    }
}
